### ISSUE TITLE
Since change to httpmanager alot of yourube links dont seem to work

### ISSUE THREAD

**Issue by MrMallenby** [kibook/pmms#21]
So upon updating to the httpmanager version of the script i noticed alot of songs that once worked did not so for testing i rolled back to the old version and confirmed the songs did infact work, included is this will be a link to 3 songs i have confirmed this issue with aswell as a screenshot of the error that comes up with the HTTP manager version. 

https://www.youtube.com/watch?v=fHAi_-DOy40

https://www.youtube.com/watch?v=GurkREc-q4I

https://www.youtube.com/watch?v=TwO0zLLybQ0
![20211105212713_1](https://user-images.githubusercontent.com/4685327/140624069-93d806da-bb7c-4c09-a672-8a5eb1bce9fe.jpg)

**Comment by kibook** [comment]
I'm guessing YouTube doesn't like that the request is coming from a plain HTTP site now, probably similar to the reason many videos get blocked on RedM which uses NUI instead of DUI.

I've reworked this in 1901e91 so pmms will now use two DUI URLs:
- For HTTPS URLs, pmms will go back to using an external page, though it has been moved to it's own repository instead of a branch on this one.
- For HTTP URLs, pmms will use the internal page provided by httpmanager by default.

I tested the YouTube videos you linked as well as the HTTP URL in #4 and they all seem to work now.
